### PR TITLE
add metaspan delegate details for kusama & polkadot

### PR DESCRIPTION
**Pull request by dcolley**

### CODE REVIEW

**Comment by dcolley** [comment]
https://kusama.subscan.io/extrinsic/19228450-4

**Comment by dcolley** [comment]
Do you need an extrinsic for polkadot too?

**Comment by stepanLav** [comment]
> Do you need an extrinsic for polkadot too?

Hello!
Yes, please, you provide accounts from different secrets, so I can't say that both is yours

**Comment by dcolley** [comment]
remark for polkadot
https://polkadot.subscan.io/extrinsic/16883802-2

File: registry/kusama.json [hunk 1]
```diff
@@ -230,5 +230,13 @@
...
         "isOrganization": false
+    },
+    {
+        "address": "HyLisujX7Cr6D7xzb6qadFdedLt8hmArB6ZVGJ6xsCUHqmx",
+        "name": "metaspan",
+        "image": "https://raw.githubusercontent.com/metaspan/metaspan.io/main/ui3/public/image/globe.png",
+        "shortDescription": "Validating on Polkadot & Kusama. Actively supporting and contributing to the community via development, utils, education, and advocacy.",
+        "longDescription": "**metaspan** is a trusted network infrastructure provider and validator on Polkadot and Kusama.\nWe are a team of experienced engineers and developers with a passion for blockchain technology and the Polkadot ecosystem.\nWe are actively involved in the community and contribute to the ecosystem via development, utilities, education, and advocacy.\nWe are committed to the long-term success of Polkadot and Kusama and are excited to be a part of the journey.",
+        "isOrganization": true
```
Comment (stepanLav) [inline registry/kusama.json:240]:
```suggestion
    },
    {
        "address": "HyLisujX7Cr6D7xzb6qadFdedLt8hmArB6ZVGJ6xsCUHqmx",
        "name": "metaspan",
        "image": "https://raw.githubusercontent.com/metaspan/metaspan.io/main/ui3/public/image/globe.png",
        "shortDescription": "Validating on Polkadot & Kusama. Actively supporting and contributing to the community via development, utils, education, and advocacy.",
        "longDescription": "**metaspan** is a trusted network infrastructure provider and validator on Polkadot and Kusama.\n\nWe are a team of experienced engineers and developers with a passion for blockchain technology and the Polkadot ecosystem.\n\nWe are actively involved in the community and contribute to the ecosystem via development, utilities, education, and advocacy.\n\nWe are committed to the long-term success of Polkadot and Kusama and are excited to be a part of the journey.",
        "isOrganization": true
    }
```

File: registry/kusama.json [hunk 2]
```diff
@@ -230,5 +230,13 @@
...
     }
```
Comment (stepanLav) [inline registry/kusama.json:241]:
```suggestion
    },
    {
        "address": "HyLisujX7Cr6D7xzb6qadFdedLt8hmArB6ZVGJ6xsCUHqmx",
        "name": "metaspan",
        "image": "https://raw.githubusercontent.com/metaspan/metaspan.io/main/ui3/public/image/globe.png",
        "shortDescription": "Validating on Polkadot & Kusama. Actively supporting and contributing to the community via development, utils, education, and advocacy.",
        "longDescription": "**metaspan** is a trusted network infrastructure provider and validator on Polkadot and Kusama.\n\nWe are a team of experienced engineers and developers with a passion for blockchain technology and the Polkadot ecosystem.\n\nWe are actively involved in the community and contribute to the ecosystem via development, utilities, education, and advocacy.\n\nWe are committed to the long-term success of Polkadot and Kusama and are excited to be a part of the journey.",
        "isOrganization": true
    }
```

Comment (stepanLav) [reply registry/kusama.json:241]: I suggest to improve a bit your formatting

File: registry/polkadot.json [hunk 3]
```diff
@@ -270,5 +270,13 @@
...
-    }
+    },
+    {
+      "address": "16ce9zrmiuAtdi9qv1tuiQ1RC1xR6y6NgnBcRtMoQeAobqpZ",
+      "name": "metaspan",
+      "image": "https://raw.githubusercontent.com/metaspan/metaspan.io/main/ui3/public/image/globe.png",
+      "shortDescription": "Validating on Polkadot & Kusama. Actively supporting and contributing to the community via development, utils, education, and advocacy.",
+      "longDescription": "**metaspan** is a trusted network infrastructure provider and validator on Polkadot and Kusama.\nWe are a team of experienced engineers and developers with a passion for blockchain technology and the Polkadot ecosystem.\nWe are actively involved in the community and contribute to the ecosystem via development, utilities, education, and advocacy.\nWe are committed to the long-term success of Polkadot and Kusama and are excited to be a part of the journey.",
+      "isOrganization": true
+  }
```
Comment (stepanLav) [inline registry/polkadot.json:281]:
```suggestion
    },
    {
        "address": "16ce9zrmiuAtdi9qv1tuiQ1RC1xR6y6NgnBcRtMoQeAobqpZ",
        "name": "metaspan",
        "image": "https://raw.githubusercontent.com/metaspan/metaspan.io/main/ui3/public/image/globe.png",
        "shortDescription": "Validating on Polkadot & Kusama. Actively supporting and contributing to the community via development, utils, education, and advocacy.",
        "longDescription": "**metaspan** is a trusted network infrastructure provider and validator on Polkadot and Kusama.\n\nWe are a team of experienced engineers and developers with a passion for blockchain technology and the Polkadot ecosystem.\n\nWe are actively involved in the community and contribute to the ecosystem via development, utilities, education, and advocacy.\n\nWe are committed to the long-term success of Polkadot and Kusama and are excited to be a part of the journey.",
        "isOrganization": true
    }
```

**Comment by stepanLav** [comment]
Hello @dcolley ,
Thank you for request!
Sorry for the delay, there is how it will looks like int the app after suggested changes:
<details>
  <summary>Screens</summary>

Screenshot 2023-09-01 at 20 34 21 4
Screenshot 2023-09-01 at 20 39 08
Screenshot 2023-09-01 at 20 39 10


</details>

**Comment by stepanLav** [comment]
Also, please, add your image to our repo in order to make sure, that our client will be able to support that, thank you!

File: registry/kusama.json (hunk 2 above)
Comment (dcolley) [reply registry/kusama.json]: this one is outdated

**Comment by dcolley** [comment]
@stepanLav is the background of the app always black/dark? If so I will load a different logo.

File: registry/kusama.json [hunk 4]
```diff
@@ -230,5 +230,13 @@
         "shortDescription": "Polkadot Moderator, Ambassador and full time Dev. I actively contribute to, advocate for, and guide the community's growth and progress.",
         "longDescription": "As a voting delegate at Kusama, I vote on proposals after deep understanding and discussions with community members. If a proposal is unclear, I abstain from voting. My aim is to ensure informed voting that benefits our community and drives progress.",
         "isOrganization": false
+    },
+    {
+        "address": "HyLisujX7Cr6D7xzb6qadFdedLt8hmArB6ZVGJ6xsCUHqmx",
+        "name": "metaspan",
+        "image": "https://raw.githubusercontent.com/metaspan/novawallet-opengov-delegate-registry/master/images/metaspan-logo-white-on-trans.png",
```
Comment (stepanLav) [inline registry/kusama.json:237]:
```suggestion
        "image": "https://raw.githubusercontent.com/novasamatech/opengov-delegate-registry/master/images/metaspan-logo-white-on-trans.png",
```

File: registry/polkadot.json [hunk 5]
```diff
@@ -270,5 +270,13 @@
         "shortDescription": "Validator Polkadot, Kusama, Cosmos | Senior Ambassador Moonbeam",
         "longDescription": "𝐔𝐓𝐒𝐀 is a trustworthy and high performance validator that supports public hotspots, relaying, and required documentation. Our team is actively working to develop local crypto communities, create a WIKI in Russian about the Polkadot and Cosmos ecosystems https://teletype.in/@lesnik13utsa. We consider Governance to be the backbone of Polkadot development and pay great attention to it. Your delegations to us = getting the right choice and free time!",
         "isOrganization": false
+    },
+    {
+        "address": "16ce9zrmiuAtdi9qv1tuiQ1RC1xR6y6NgnBcRtMoQeAobqpZ",
+        "name": "metaspan",
+        "image": "https://raw.githubusercontent.com/metaspan/novawallet-opengov-delegate-registry/master/images/metaspan-logo-white-on-trans.png",
```
Comment (stepanLav) [inline registry/polkadot.json:277]:
```suggestion
        "image": "https://raw.githubusercontent.com/novasamatech/opengov-delegate-registry/master/images/metaspan-logo-white-on-trans.png",
```

**Comment by stepanLav** [comment]
@dcolley 
sorry for the delay, this is how it will look like in the app:
<details>
  <summary>App screens</summary>

Screenshot 2023-09-25 at 09 04 29
Screenshot 2023-09-25 at 09 04 32
Screenshot 2023-09-25 at 09 04 34


</details>

**Review by stepanLav — APPROVED** [review]

**Comment by stepanLav** [comment]
@dcolley 
You have conflicts in PR, could you resolve them please?

**Comment by dcolley** [comment]
Resolved, thanks

**Comment by stepanLav** [comment]
sorry for the delay, I've created separate PR in order to merge it without conflicts, as yours has got another one 
https://github.com/novasamatech/opengov-delegate-registry/pull/81

**Pull request closed without merging**